### ISSUE TITLE
It seems to not working correctly.

### ISSUE THREAD

**Issue by sacch6579** [seancfoley/IPAddress#51]
The code below doesn't work correctly. It is suppose to return all true, but only the first line gives true others give false.

`package com.mycompany.ipaddresstests;

import inet.ipaddr.*;

public class MainTests {

  public static void main(String[] args) {
    IPAddress address = new IPAddressString("10.132.0.1/16").getAddress();

    System.out.println(address.contains(new IPAddressString("10.132.0.1").getAddress()));
    System.out.println(address.contains(new IPAddressString("10.132.1.1").getAddress()));
    System.out.println(address.contains(new IPAddressString("10.132.2.1").getAddress()));
    System.out.println(address.contains(new IPAddressString("10.132.3.1").getAddress()));
  }
}
`

**Comment by seancfoley** [comment]
Hello,

This is a duplicate of issue #26, issue #30,  issue #39, and issue #45.

This is explained in [this wiki entry](https://github.com/seancfoley/IPAddress/wiki/Code-Examples-1:-Creating,-Parsing,-Checking-Host-Address-Subnet,-Extracting-Address-Subnet-Info#parse-prefixed-ip-address-string-as-subnet-or-ip-address) and this [entry](https://github.com/seancfoley/IPAddress/wiki/Code-Examples-2:-Subnetting-and-Other-Subnet-Operations#check-if-subnet-contains-address-or-subnet) - see enclosingBlockContains() in particular.

In summary, 
10.132.0.1/16 is the same as 10.132.0.1.
So 10.132.0.1 contains 10.132.1.1 is false.

What you want is 10.132.0.0/16, which is a block of addresses from 10.132.0.0 to 10.132.255.255/16.
10.132.0.0/16 contains 10.132.1.1  is true.
